lib/getStatistics.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+var Cesium = require('cesium');
+var defaultValue = Cesium.defaultValue;
+var DeveloperError = Cesium.DeveloperError;
+var isDataUri = require('./isDataUri');
+var PrimitiveHelpers = require('./PrimitiveHelpers');
+
+var defined = Cesium.defined;
+
+module.exports = getStatistics;
+
+/**
+ * Creates an object type for glTF Statistics.
+ *
+ * @property {String} gltfPath The path to the glTF file.
+ * @property {Number} buffersSizeInBytes The total size of all the buffers.
+ * @property {Number} numberOfImages The number of images in asset.
+ * @property {Number} numberOfExternalRequests The number of external requests required to fetch the asset data.
+ * @property {Number} numberOfDrawCalls The number of draw calls required to render the asset.
+ * @property {Number} numberOfRenderedPrimitives The total number of rendered primitives in the asset.
+ * @property {Number} numberOfNodes The total number of nodes in the asset.
+ * @property {Number} numberOfMeshes The total number of meshes in the asset.
+ * @property {Number} numberOfMaterials The total number of materials in the asset.
+ * @property {Number} numberOfAnimations The total number of animations in the asset.
+ *
+ * @constructor
+ *
+ * @see getStatistics
+ * @see printStatistics
+ */
+function Statistics() {
+    this.gltfPath = '';
```

Comment:
I think we can take this out of Statistics entirely. Before printing the stats `Pipeline` can print `Statistics for <input path>`. Sorry for not catching this in the previous review.